index.js

```diff
@@ -227,6 +232,16 @@ const ContainerService = {
           sendReply(this[message.method](message));
         }
       });
+
+      // This is only used for theme notifications
+      api.browser.runtime.onConnect.addListener((port) => {
```

Comment:
If not a new object, at least group these methods together so all the theme-managing logic is together.